package.json

```diff
@@ -33,6 +33,8 @@
     "aos": "^2.3.4",
     "cleave.js": "^1.5.8",
     "d3": "^5.15.0",
+    "jquery": "^3.4.1",
```

Comment:
I am afraid this will affect the original jquery package since you may load the jquery twice. One from original `wordpress` and one from here.

Reply:
Hi @hi-upchen  
Will take a look again.
I couldn't find the original jquery and can you advise how to import jquery plugin without import it again. It's now showing `$ is not a function`.